jccli/jc_api_v1.py

```diff
@@ -37,7 +37,7 @@ def retrieve_users(self, user_ids=[]):
         Retrieve a list of users corresponding to ids
         """
         # FIXME: This is not an ideal way to do this, but search_systemusers_post doesn't seem to allow filtering on ID
-        all_users = self.get_users()
```

Comment:
looks like the jc_api_v1.get_user method can be simplified by using search API as well?